rasa/core/training/interactive.py

```diff
@@ -579,6 +579,18 @@ def _slot_history(tracker_dump: Dict[Text, Any]) -> List[Text]:
     return slot_strings
 
 
+async def _retry_on_error(func: Callable, export_path: Text, *args, **kwargs) -> None:
+    while True:
+        try:
+            return await func(export_path, *args, **kwargs)
```

Comment:
Why do we need the `await` here? `_write_stories_to_file`, `_write_nlu_to_file` and `_write_domain_to_file` are all synchronous

Reply:
Good question. I removed it.